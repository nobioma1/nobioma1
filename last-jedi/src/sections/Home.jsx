import React from 'react';
import ScrollAnimation from 'react-animate-on-scroll';

import Intro from '../components/Intro';
import Layout from '../components/Layout';

const Home = () => {
  return (
    <Layout centerX centerY fullHeight id="home">
      <ScrollAnimation animateIn="zoomIn" animateOnce>
        <Intro />
      </ScrollAnimation>
    </Layout>
  );
};

export default Home;
