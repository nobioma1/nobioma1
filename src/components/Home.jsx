import React from 'react';
import { HomeContainer, Hello, ImageContainer, Info } from '../styles';
import avatar from '../assets/avatar.jpg';

const Home = () => {
  return (
    <HomeContainer>
      <Info>
        <Hello>
          <p>Hello, I am</p>
        </Hello>
        <h1>Noble Obioma</h1>
        <p>Software Engineer</p>
      </Info>
      <ImageContainer>
        <img src={avatar} alt="Noble Obioma's Avatar" />
      </ImageContainer>
    </HomeContainer>
  );
};

export default Home;
