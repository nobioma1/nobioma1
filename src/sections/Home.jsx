import React from 'react';
import Intro from '../components/Intro';
import Layout from '../components/Layout';

const Home = () => {
  return (
    <Layout centerX centerY fullHeight id="home">
      <Intro />
    </Layout>
  );
};

export default Home;
