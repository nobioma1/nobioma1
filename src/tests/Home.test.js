import React from 'react';
import renderer from 'react-test-renderer';

import Home from '../components/Home';

describe('Home', () => {
  it('Home Should Match Snapshot', () => {
    const tree = renderer.create(<Home />);

    expect(tree).toMatchSnapshot();
  });
});
