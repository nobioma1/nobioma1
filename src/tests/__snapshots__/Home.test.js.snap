// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Home Should Match Snapshot 1`] = `
<div
  className="sc-gzVnrw fxpRuc"
>
  <div
    className="sc-dnqmqq kChQzR"
  >
    <div
      className="sc-htoDjs tbauv"
    >
      <p>
        Hello, I am
      </p>
    </div>
    <h1>
      Noble Obioma
    </h1>
    <p>
      Software Engineer
    </p>
  </div>
  <div
    className="sc-bdVaJa cdJaUk"
  >
    <img
      alt="Noble Obioma's Avatar"
      src="avatar.jpg"
    />
  </div>
</div>
`;
